autoPyTorch/pipeline/components/training/data_loader/base_data_loader.py

```diff
@@ -57,7 +57,8 @@ def __init__(self, batch_size: int = 64,
         self.add_fit_requirements([
             FitRequirement("split_id", (int,), user_defined=True, dataset_property=False),
             FitRequirement("Backend", (Backend,), user_defined=True, dataset_property=False),
-            FitRequirement("is_small_preprocess", (bool,), user_defined=True, dataset_property=True)])
+            # FitRequirement("is_small_preprocess", (bool,), user_defined=True, dataset_property=True)
```

Comment:
I do not know what this `is_small_preprocess` does, but can we remove it if we will not use it anymore?